instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './PostContainer.scss'
+import CommentSection from '../CommentSection/CommentSection'
+import PropTypes from 'prop-types'
+
+const PostContainer = props => {
+    console.log('PostContainer props', props)
+    return (
+        <div className="container post-container">
+        <div className="row post-row">
+            <div className="col post-content">
+                Username: <span>{props.post.username}</span>
+            </div>
+            <div className="col post-content">
+                <img src={props.post.imageUrl} ref={props.username}></img>
+
+            </div>
+            <div className="col">
+            {props.post.comments.map(comment => {
+             return <CommentSection comment={comment} />
+            })}
+
+            </div>
+        </div>
+        </div>
+
+    );
+}
+
+PostContainer.propTypes = {
```

Comment:
Awesome to see you've already implemented propType! Please keep doing this, it gives you a level of certainty when working with your data which in a bigger application is extremely important!